ReactSkia/pluginfactory/plugins/stub/PlatformManager.h

```diff
@@ -14,6 +14,11 @@ namespace rnsplugin {
 
 using namespace std;
 
+using DeviceCPUInfo = RNSPlatformManagerInterface::PlatformDevice::DeviceProcessUintInfo;
+using DeviceGPUInfo = RNSPlatformManagerInterface::PlatformDevice::DeviceProcessUintInfo;
+using DeviceOsInfo = RNSPlatformManagerInterface::PlatformDevice::DeviceOsInfo;
```

Comment:
Will it be more right to name as DeviceOSInfo ?? If you are changing, change it all over

Reply:
Other APIs also using camel case so we are using same for Os